documentation/src/main/wiki/LayoutComparator.md

```diff
@@ -10,7 +10,10 @@ Resource name: screen
 
 ##### Parameters
 
-No parameters
+| Parameter | Value | Description | Mandatory |
+| --------- | ----- | ----------- | --------- |
+| `pixelThreshold` | int (bigger than 0) | The value to set the error threshold in pixels | no |
+| `percentageThreshold` | double (0 to 100) | The value to set the error threshold in percentages | no |
```

Comment:
This documentation is missing some details - how the threshold mechanism works and what will happen on report if the difference will be below/equal/above threshold?